docs/source/ruletypes.rst

```diff
@@ -2780,6 +2780,70 @@ Example With SMS usage::
     twilio_auth_token: "abcdefghijklmnopqrstuvwxyz012345"
     twilio_account_sid: "ABCDEFGHIJKLMNOPQRSTUVWXYZ01234567"
 
+Wechat
+~~~~~~~~
+
+Wechat will send notification with a application. The body of the notification is formatted the same as with other alerters. See: https://work.weixin.qq.com/api/doc/90000/90135/90236
+
+Required:
+
+``wechat_corp_id``:  Wechat corp id.
+
+``wechat_agent_id``: The wechat agent id you are going to send the message.
+
+``wechat_secret``:  The wechat agent secret.
+
+Optional:
+
+``wechat_to_party`` & ``wechat_to_user`` & ``wechat_to_tag``: There must be one.
```

Comment:
I would move this to the `Required` section and rephrase to:

'One of `wechat_to_party`, `wechat_to_user`, or `wechat_to_tag`'